### PR TITLE
Add ZenHub Definitions

### PR DESCRIPTION
**Pull request by djmaster458**
## Overview
I added some ZenHub definitions

### CODE REVIEW

File: src/definitions.md [hunk 1]
```diff
@@ -21,3 +21,6 @@ Depending on the files changes and timings, Git will either automatically handle
 ### Pull Request
 A pull request is a feature of GitHub that allows your feature branch to be reviewed before merging into another branch.
 This allows for developers to review each others' code for bugs or missing specifications.
+
```
Comment (djmaster458) [inline src/definitions.md:24]: why is there whitespace

**Comment by tjcanro** [comment]
bruh

File: src/definitions.md (hunk 1 above)
Comment (MarcosC1214) [reply src/definitions.md:24]: What is a ZenHub?

File: src/definitions.md [hunk 2]
```diff
@@ -21,3 +21,6 @@ Depending on the files changes and timings, Git will either automatically handle
...
+### ZenHub Extension
+We use this for agile
```
Comment (DavidLangner) [inline src/definitions.md:26]: I appreciate the lower case "agile" in this case. lol

File: src/definitions.md (hunk 1 above)
Comment (djmaster458) [reply src/definitions.md:24]: Zenhub is a great extension for GitHub that allows for us to track projects, issues, etc...
You'll get a taste of it I believe with an Agile development class later on

**Pull request closed without merging**